app/src/Controller/ArticlesController.php

```diff
@@ -59,13 +63,11 @@ public function view($slug = null)
     public function add()
     {
         $article = $this->Articles->newEmptyEntity();
+        $this->Authorization->authorize($article);
         if ($this->request->is('post')) {
             $article = $this->Articles->patchEntity($article, $this->request->getData());
 
-            // @TODO: Hardcoding the user_id is temporary and will be removed later
-            if (property_exists($article, 'user_id')) {
-                $article->user_id = 1;
-            }
```

Comment:
We removed the TODO part with new code. However, we still have the TODO here  https://github.com/sergio9977/tutorial-cakephp/blob/55f47bb0cb26cdc357529117aaed34e3a5679147/app/templates/Articles/add.php#L20-L21

Please check.